src/Symfony/Component/Serializer/Mapping/ClassMetadata.php

```diff
@@ -28,7 +28,7 @@ class ClassMetadata implements ClassMetadataInterface
     public $name;
 
     /**
-     * @var AttributeMetadataInterface[]
+     * @var array
```

Comment:
Because the key is a `string` and some tools like Scrutinizr consider that the `[]` notation is only applicable for packed arrays (consecutive `int` as keys).